src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralWhen.java

```diff
@@ -0,0 +1,31 @@
+/*
+WhitespaceAfter
+
```

Comment:
Same as other PR. Show examples with new lines, and show enable/disable of property `allowLineBreaks`.

Reply:
done. 

>  enable/disable of property allowLineBreaks.

what property?

Reply:
We have [allowLineBreaks](https://checkstyle.sourceforge.io/checks/whitespace/nowhitespacebefore.html#Properties) in NoWhitespaceBefore check but we don't have it for WhitespaceAfter check.

Reply:
ok, I may have mixed up PRs or documentation.